README.md

```diff
@@ -16,11 +16,12 @@ The config object passed in must have the following sections and fields:
 ```js
 var config = {
   secret: 'The client secret, only needed for OAuth v1',
-  clientId: 'Your own client id',
+  clientId: 'Your own client id, only needed for OAuth v1',
   domain: 'The token issuer w/o https:// or the trailing /',
   realm: 'The realm of the token, used in the challenge headers, full url (w/ https:// and  trailing /)',
   jwksUrl: 'The url to retrieve the jwk from, only needed for OAuth v2',
   audience: 'The audience for the auth token',
+  disableV1: 'Should auth v1 be enabled (defaults to false), not required',

```

Comment:
It is opt-in, in terms of you opt-in to disable v1.

Can't do "you opt-in to enable v1" or it'll break existing clients.

I should fix the wording in the readme though, cause it says the opposite